llvm/include/llvm/Transforms/Utils/ExtraPassManager.h

```diff
@@ -0,0 +1,89 @@
+//===- ExtraFunctionPassManager.h - Run Optimizations on Demand ---------*- C++
+//-*-===//
```

Comment:
Possible to avoid the odd line break here?

Reply:
Argh, I thought I already fixed this up....